uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/loyalty/LoyaltyFeatureFlags.kt

```diff
@@ -0,0 +1,7 @@
+package com.karhoo.uisdk.screen.booking.checkout.loyalty
+
+object LoyaltyFeatureFlags {
+    var loyaltyEnabled = true
+    var loyaltyCanBurn = true
```

Comment:
If you change the `loyaltyCanEarn` and `loyaltyCanBurn` values to `loyaltyEnabled && true` we'll be able to disable both just by disabling `loyaltyEnabled`. And it would be the same logic as on iOS

Reply:
Agree, but there is only one usage of the canEarn and canBurn and is way after the check for enabled so basically, if the enabled one is false, the canEarn and canBurn will never be reached anyway